### DISCUSSION TITLE
Animation preview for a music video

### DISCUSSION THREAD

**Discussion by Cjjuntti** [keiwando/toonsquid#573] [Show and tell]
![IMG_0364](https://github.com/keiwando/toonsquid/assets/120066598/8845feca-9495-4d33-a4f5-78c847ba5193)

**Comment by TempleLionStudios** [comment]
Want feedback?

**Reply by Cjjuntti** [reply]
I would love to know what you think so far. It's probably gonna be my biggest toonsquid project yet.

**Reply by TempleLionStudios** [reply]
Well first of all i salute you for makin a project! 🙏🏽⭐️

The thing that struck me the most isthat from the side the head needs to be less flat. Look at head anatomy, the head goes back quite a lot 😊

But good luck on your journey!

**Reply by Cjjuntti** [reply]
Head turns are hard, but I at least tried with it. This is only a small portion of what I’m working on. When I finish the full animation, I’d like to see what you think of the full video when it’s done.

**Reply by TempleLionStudios** [reply]
I will gladly look at what you make 😊🙏🏽

Keep up the great work! 🥳⭐️
